src/aims_ui/page_helpers/error/error_utils.py

```diff
@@ -93,12 +93,27 @@ def error_page_xml(page_name, user_input):
       page_name,
       'XML Attack Detected. This incident will be reported.',
       [
-          'If you beleive this is an eroneous detection, please contact the AIMS team using the link at the bottom of the page.',
+          'If you believe this is an erroneous detection, please contact the AIMS team using the link at the bottom of the page.',
           'You can also try removing special characters from the address and try again.',
       ],
   )
 
 
+def error_page_too_many_jobs(page_name, user_input,job_count, max_jobs):
+  """ Return error page for too many BM jobs attack """
```

Comment:
I don't think "attack" is correct here, more like "return error page if too many bulk jobs are already in progress"